src/Compilers/Core/Portable/Operations/ICompoundAssignmentExpression.cs

```diff
@@ -16,10 +14,16 @@ public interface ICompoundAssignmentExpression : IAssignmentExpression, IHasOper
         /// <summary>
         /// Kind of binary operation.
         /// </summary>
-        BinaryOperationKind BinaryOperationKind { get; }
+        BinaryOperatorKind OperatorKind { get; }
+
         /// <summary>
         /// <code>true</code> if this assignment contains a 'lifted' binary operation.
         /// </summary>
         bool IsLifted { get; }
+
+        /// <summary>
+        /// <code>true</code> if overflow checking is performed for the arithmetic operation.
+        /// </summary>
+        bool IsChecked { get; }
```

Comment:
>bool IsChecked { get; } [](start = 8, length = 23)

Do we have positive and negative tests for this property?

Reply:
Done.